traffic_ops/testing/api/v5/origins_test.go

```diff
@@ -630,12 +633,14 @@ func GetOriginID(t *testing.T, name string) func() int {
 		assert.RequireNoError(t, err, "Get Origins Request failed with error:", err)
 		assert.RequireEqual(t, 1, len(origins.Response), "Expected response object length 1, but got %d", len(origins.Response))
 		assert.RequireNotNil(t, origins.Response[0].ID, "Expected ID to not be nil.")
-		return *origins.Response[0].ID
+		return origins.Response[0].ID
 	}
 }
 
 func CreateTestOrigins(t *testing.T) {
 	for _, origin := range testData.Origins {
+		origin.TenantID = GetTenantID(t, origin.Tenant)()
+		origin.DeliveryServiceID = GetDeliveryServiceId(t, origin.DeliveryService)()
```

Comment:
with the changes you have made to the client, this should no longer be necessary